### DISCUSSION TITLE
Support satellite community?

### DISCUSSION THREAD

**Discussion by Chris230291** [iptv-org/community#12] [Ideas]
Would you consider adding an additional field to the database for "service references" so that the satellite community can easily use your grabbers?
Here is an example service reference for Sky Sports Premier league UK `1:0:1:14B7:7E7:2:11A0000:0:0:0:`
The field should accept lists of values like the example given.
Doing this would allow us to generate a file like this one: https://raw.githubusercontent.com/doglover3920/EPGimport-Sources/main/rytec.channels-sat.xml

**Comment by BellezaEmporium** [comment]
Some other repos actually do this, see the HANSSettings repository : https://github.com/technl/Hanssettings

However, it may add a level of complexity to our already quite complex structure. Matching EPG with a service reference might be more precise, but if that parameter begins to change, it'll be a bit more difficult to pin this out rather than a TVG-id.

**Comment by Chris230291** [comment]
That appears to just be bouquets.

Right now we could create a file that links iptv-org's id's to service references and host it externally from this project.
Including this info in the database would marry service references to channels, meaning any changes done to channel ID's here wouldn't break links to service references. We can just generate a new map file from the database.
